gensim/models/fasttext.py

```diff
@@ -89,12 +95,10 @@ def __init__(self, sentences=None, sg=0, hs=0, size=100, alpha=0.025, window=5,
         if self.word_ngrams <= 1 and self.max_n == 0:
             self.bucket = 0
 
-        super(FastText, self).__init__(
-            sentences=sentences, size=size, alpha=alpha, window=window, min_count=min_count,
+        super(FastText, self).__init__(sentences=sentences, size=size, alpha=alpha, window=window, min_count=min_count,
```

Comment:
PEP8: Arguments on first line forbidden when not using vertical alignment.
https://www.python.org/dev/peps/pep-0008/#indentation